packages/report-e2e-tests/src/examples/axe-results-without-issues.snap.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en"
   ><head
     ><meta charset="UTF-8" /><title
-      >Accessibility Insights automated checks result</title
+      >Accessibility Insights FastPass results</title
```

Comment:
I think it's probably not ideal for these snapshots to need updates; they're for cases from the report package that really are only running automated checks, not a full FastPass. Would probably be preferable to either parameterize or separate components to allow these the report package cases to stay as is

Reply:
Good call, I've parameterized it!